rebound/librebound.py

```diff
@@ -215,7 +294,11 @@ def add(self, particle=None, **kwargs):
                 for p in particle:
                     self.add(p)
             elif isinstance(particle,str):
+                for key, unit in self.units.items(): # check if units aren't set to set defaults
```

Comment:
Thanks!  I couldn't find a good way to express it since it was a dictionary, but your suggestion made it clear:  if None in self.units.values()